README.md

```diff
@@ -2,41 +2,37 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-An api that returns double-dummy results for a given board.
+An api that returns double-dummy results for a given deal.
 
 ## Usage
 ```
 curl --header "Content-Type: application/json" --request POST --data '{"hands":{"S":["D3", "C6", "DT", "D8", "DJ", "D6", "CA", "C3", "S2", "C2", "C4", "S9", "S7"],"W":["DA", "S4", "HT", "C5", "D4", "D7", "S6", "S3", "DK", "CT", "D2", "SK","H8"],"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"]}}' https://dds.prod.globalbridge.app/api/dds-table/
 ```
-Uses Bo Hagland's solver https://github.com/dds-bridge/dds -- requires the libdds.so (or dds.dll in windows) to be installed and accessible.
-Credit to Alexis Rimbaud of NukkAI for the python dds wrapper.
-
-
-## Build the C++ library for local testing
-
+or equivalently
 ```
-make libdds-build
+make curl_prod
```

Comment:
How about only using curl_local target in documentation?

I don't think testing remote service is as useful for someone wanting to make change to the service. Testing production environment is more useful for someone trying to using the API. But API documentation is separate from the README.

Reply:
I think it's useful for the user to be able to see exactly what happens when running against prod so that he can compare it to the local version. It also helps make it clear that there are distinct local and remote servers - not everyone is used to this kind of development.

Have we written the API documentation? If so I could link to it.

Reply:
There is GitHub.io version up. But future version might move to a cloud bucket. More can be found from discord discussions.